nodestream/pipeline/class_loader.py

```diff
@@ -57,9 +59,10 @@ def find_class_initializer(self, implementation, factory=None):
 
     def load_class(self, implementation, arguments=None, factory=None):
         arguments = arguments or {}
+        resolved_arguments = LazyLoadedArgument.resolve_if_needed(arguments)
```

Comment:
Maybe this should move back out to the file loader to keep the responsibility out of  `class_loader`. Class loader technically doesn't know anything about yaml. what do you think?

Reply:
Yeah that does make more sense